fe/fe-core/src/main/java/com/starrocks/common/proc/ExternalDbsProcDir.java

```diff
@@ -37,18 +39,22 @@ public ProcResult fetchResult() {
             return result;
         }
 
-        // get info
-        List<List<Comparable>> dbInfos = new ArrayList<List<Comparable>>();
+        List<List<Comparable>> dbInfos = new ArrayList<>();
+
+
+        if (!Config.disable_check_db_status) {
```

Comment:
I think we can set a threshold, if the numbers exceed the threshold, we skip it directly. Then we don't need this configuration.

Reply:
The execution logic of different catalogs is different. Will it make users more confused? All catalogs keep the same logic, check or not check, maybe a better choice